Client.xcodeproj/Brave.xctestplan

```diff
@@ -28,13 +28,10 @@
   },
   "testTargets" : [
     {
-      "skippedTests" : [
-        "NavigationRouterTests\/testNavigationPathInPrivateBrowsingOnlyMode()"
-      ],
       "target" : {
         "containerPath" : "container:Client.xcodeproj",
-        "identifier" : "F84B21D21A090F8100AAB793",
-        "name" : "ClientTests"
+        "identifier" : "5D1DC51A20AC9AFA00905E5A",
+        "name" : "DataTests"
```

Comment:
Not a bugfix: I changed order of tests so tests without host app: DataTests, SharedTests are ran first